dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java

```diff
@@ -28,6 +28,8 @@
 @ConfigurationProperties("master")
 public class MasterConfig {
     private int listenPort;
+    private int masterFetchCommandNum;
+    private int masterPreExecThreads;
```

Comment:
What about stripping the prefix `master` because all configs in this section are all `master` config?

Reply:
OK, I will fix it.